docs/resources/aws_shield_subscription.md

```diff
@@ -0,0 +1,55 @@
+---
+title: About the aws_shield_subscription Resource
+platform: aws
+---
+
+# aws\_shield\_subscription
+
+Use the `aws_shield_subscription` InSpec audit resource to test properties of a shield subscription.
+
+## Syntax
+
+ An `aws_shield_subscription` resource block uses the parameter to return a shield subscription.
+
+    describe aws_shield_subscription do
+      it { should exist }
+    end
+
+
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on Shield Subscriptions](https://docs.aws.amazon.com/waf/latest/developerguide/shield-chapter.html).
+
+
+## Properties
+
+|Property                      | Description|
+| ---                          | --- |
+|auto\_renew                   | If ENABLED, the subscription will be automatically renewed at the end of the existing subscription period. AutoRenew can be either ENABLED or DISABLED. |
+|end\_time                     | The date and time your subscription will end. |
+|limits                        | Specifies how many protections of a given type you can create. |
+|proactive\_engagement\_status | If ENABLED, the DDoS Response Team (DRT) will use email and phone to notify contacts about escalations to the DRT and to initiate proactive customer support. <br/> If PENDING, you have requested proactive engagement and the request is pending. The status changes to ENABLED when your request is fully processed. <br/> If DISABLED, the DRT will not proactively notify contacts about escalations or to initiate proactive customer support. |
+|start\_time                   | The start time of the subscription, in Unix time in seconds. |
+|time\_commitment\_in\_seconds | The length, in seconds, of the AWS Shield Advanced subscription for the account. |
+
+For a comprehensive list of properties available, see [the API reference documentation](https://docs.aws.amazon.com/waf/latest/DDOSAPIReference/API_Subscription.html)
+
+## Examples
+
+##### Check Auto Renew of a Shield Subscription
+
+    describe aws_shield_subscription do
+      its('auto_renew')  { should eq 'ENABLED' }
+    end
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list of available matchers, please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+## AWS Permissions
+
+Your [Principal](https://docs.aws.amazon.com/IAM/latest/UserGuide/intro-structure.html#intro-structure-principal) will need the `shield:DescribeSubscription` action with Effect set to Allow.
+
+You can find detailed documentation at [Actions, Resources, and Condition Keys for Amazon RDS](https://docs.aws.amazon.com/waf/latest/developerguide/shield-chapter.html).
```

Comment:
The text and the URL should be updated.
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awsshield.html

Reply:
updated, thanks!